ios/Podfile

```diff
@@ -1,7 +1,9 @@
 require_relative 'podspec_versions.rb'
 
-platform :ios, '11.3'
+platformVersion = '11.3'
+platform :ios, platformVersion
 
+inhibit_all_warnings!
```

Comment:
Est-ce qu'on ne devrait pas laisser les inhibit warnings par pod. IMO c'est pertinant de voir les warning des certains pod, surtout un pod qu'on vient d'Ajouter. En mettant `inhibit_all_warnings` on le les verra jamais non ?

Reply:
Can you write in english in our open source repos ?

Moerover, I tend to agree with Marc here that our warning removals should be by pods and not overall.

Reply:
Good, agreed! :)

I'll revert this change.